src/experiments/webgpuaudio/main.js

```diff
@@ -13,6 +13,10 @@ let toggleButton = null;
 let audioContext = null;
 let isPlaying = false;
 
+// Change this value to true to run tests and see test errors on JS console
+// during audio processing. Not intended for production, only for testing.
+let runTests = false;
```

Comment:
How about having `TEST_MODE = false` in `constants.js` and use it by importing the file everywhere?

Reply:
Then we don't have to pass it around via postMessage and alike.

Reply:
Good idea, done!